wg-meetings/2021-05-19.md

```diff
@@ -0,0 +1,41 @@
+# Node.js  Diagnostics WorkGroup Meeting 2021-05-19
+
+## Links
+
+* **Recording**: https://www.youtube.com/watch?v=SoezTqnkBRk
+* **GitHub Issue**: https://github.com/nodejs/diagnostics/issues/477
+
+## Present
+
+* Michael Dawson (@mhdawson)
+* Rafael Gonzaga (@rafaelgss)
+* Gerhard Stöbich (@flarna)
+* Chengzhong Wu (@legendecas)
+* Gireesh Punathil (@gireeshpunathil)
+
+## Agenda
+
+## Announcements
+
+*Extracted from **diag-agenda** labelled issues and pull requests from the **nodejs org** prior to the meeting.
+
+### nodejs/node
+
+* async_hooks: stabilize part of AsyncLocalStorage [#37675](https://github.com/nodejs/node/pull/37675)
+  * Require some doc changes
+  * Michael will volunteer to help do the final edits.
+
+### nodejs/diagnostics
+
+* Identify async_hooks use cases beyond AsyncLocalStorage [#437](https://github.com/nodejs/diagnostics/issues/437)
+  * Next step is to publish a blog post.
+
+* WG also reviewed the Best practice documentation presentation that is slated to be presented in the openjs conference.
```

Comment:
I'm interested in this presentation. Is there a preview or what time is the conference?

Reply:
sent you a mail with the details.